src/qibo/gates/gates.py

```diff
@@ -455,12 +455,22 @@ def qasm_label(self):
 
 
 class Align(Gate):
-    def __init__(self, *q):
+    """Aligns proceeding qubit operations and (optionally) waits ``delay`` amount of time.
+
+    Args:
+        *q (int): The qubit ID numbers.
+        delay (int): (optional) The time (in ns) for which to delay circuit execution on the specified qubits.
+    """
+
+    def __init__(self, *q, delay=0):
```

Comment:
You can also add a new attribute `self.delay = delay` for this gate. This would make qiboteam/qibolab#541 simpler as you could directly do `gate.delay` to get the delay, instead of some weird indexing of `init_args`/`init_kwargs` (which are probably not a very good mechanism in the first place).

Reply:
Implemented, and updated qiboteam/qibolab#541 accordingly. Thank you.